backend/worker.py

```diff
@@ -344,31 +347,39 @@ def post(self):
 class TransferAdminPermissionsHandler(BaseHandler):
     """Handler of transfer admin permissions."""
 
-    def add_permissions(self, user, permissions):
+    def add_permissions(self, user, institution):
         """
         This method adds new permissions in user with the permissions passed in parameter.
```

Comment:
permissions passed as a parameter? It's showing here that you're passing institution instead of the previous parameter, permissions.